src/libslic3r/PrintConfig.cpp

```diff
@@ -3860,6 +3860,13 @@ void PrintConfigDef::init_fff_params()
     def->min = 1;
     def->set_default_value(new ConfigOptionInt(1));
 
+    def = this->add("enforce_first_layer_min_bead_width", coBool);
```

Comment:
Can we make this new parameter a line width parameter similar to min_bead_width? e.g.  `initial_layer_min_bead_width`

Reply:
Good idea. Let me update a new patch.